### PR TITLE
Release v4061

### PR DESCRIPTION
**Pull request by ghzlatarev**
## Description

* Unfilter `external_propose_majority` extrinsic
* Bump spec_version

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [x] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari`, `A-dolphin` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

File: runtime/dolphin/src/lib.rs [hunk 1]
```diff
@@ -125,7 +125,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("dolphin"),
     impl_name: create_runtime_str!("dolphin"),
     authoring_version: 2,
-    spec_version: 4060,
+    spec_version: 4061,
```
Comment (ghzlatarev) [inline runtime/dolphin/src/lib.rs:128]: not sure we need to keep these in sync anymore

Comment (ferrell-code) [reply runtime/dolphin/src/lib.rs:128]: Yeah doesn't really matter anymore but I would still think keeping them synced makes the most sense

**Review by ferrell-code — COMMENTED** [review]
Looks good, lets run CI to update change log too

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,10 +6,12 @@
 
 ### Changed
 - [\#1071](https://github.com/Manta-Network/Manta/pull/1071) Charge 0-asset XCM instructions as if they were 1-asset [MACADO]
+- [\#1084](https://github.com/Manta-Network/Manta/pull/1084) Use safe-math in xcm `buy_weight` impl [MACA]
```
Comment (ferrell-code) [inline CHANGELOG.md]: Maybe manually update changelog? cause generator just adds it to 4.0.6.

**Review by ferrell-code — APPROVED** [review]

**Comment by ghzlatarev** [comment]
https://github.com/Manta-Network/Manta/compare/release-v4.0.5-1?expand=1

**Pull request closed without merging**